manifests/init.pp

```diff
@@ -5,6 +5,8 @@
 # === Parameters
 #
 # $root:: Configures the root directory for the TFTP server
+# $package:: name of the tftp package
+# $syslinux_package:: essential for pxe boot
```

Comment:
A more useful description would be good, e.g. the name of the package providing SYSLINUX